### PR TITLE
Feature/product modal edit

### LINKED ISSUE #6: Implement [ProductModalEdit] screen ui
[Issuehunt badges](https://issuehunt.io/r/dooboolab/whatssub-admin/issues/6)

image

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $10.00)

- [hyochan hyochan](https://issuehunt.io/u/hyochan) ($10.00)

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/dooboolab/whatssub-admin/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by YongPilMoon**
## Description

I implemented Product Modal and some components for it. (Modal, Cell, Input, Select)
예시
## Related Issues
Closes #6

## Tests
I added the following tests:

Component Test(Modal, Cell, Input, Select)

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/whatssub/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by hyochan** [comment]
@geoseong @marsinearth @godon019 Please review this wonderful people!

File: src/components/ui/ProductModal.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+interface ProductModalProps {
+  isVisible: boolean;
+  onClose: () => void;
+}
+
+const modalStyle = css`
+  padding: 40px;
+  width: 720px;
+  background-color: #f8f9fa;
```
Comment (geoseong) [inline src/components/ui/ProductModal.tsx]: I think style with changing theme is not implemented.
changing that codes allow this to change style with theme change
```
background: ${(props) => props.theme.background};
color: ${(props) => props.theme.fontColor};
```

File: src/components/ui/ProductModal.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  return (
+    <Modal isVisible={isVisible} onClose={onClose} customModalCss={modalStyle} isDisabled={true}>
+      <HeaderDiv>
+        <TitleH1>상품추가</TitleH1>
+        <ImageWrapper onClick={onClose}>
+          <ClearImg src={IC_CLEAR} alt="취소"/>
+          <Ink/>
+        </ImageWrapper>
+      </HeaderDiv>
+      <Formik
```
Comment (geoseong) [inline src/components/ui/ProductModal.tsx]: Did you intentionally avoid `validationSchema` prop?
this formik form doesn't have any validation.

File: src/components/ui/ProductModal.tsx (hunk 1 above)
Comment (YongPilMoon) [reply src/components/ui/ProductModal.tsx]: Thanks for your kind example.

File: src/components/ui/ProductModal.tsx (hunk 2 above)
Comment (YongPilMoon) [reply src/components/ui/ProductModal.tsx]: No. It's because I am lazy... I will do it. Thanks. 🙏

**Pull request merged**